model.py

```diff
@@ -6409,6 +6409,18 @@ class CirculationEvent(Base):
     TYPE = u"event"
 
     # The names of the circulation events we recognize.
+    # They may be sent to third-party analytics services
+    # as well as used locally.
+
+    # Events that happen in a circulation manager.
+    NEW_PATRON = u"circulation_manager_new_patron"
+    CM_CHECKOUT = u"circulation_manager_check_out"
+    CM_CHECKIN = u"circulation_manager_check_in"
+    CM_HOLD_PLACE = u"circulation_manager_hold_place"
+    CM_HOLD_RELEASE = u"circulation_manager_hold_release"
+    CM_FULFILL = u"circulation_manager_fulfill"
+
+    # Events that we hear about from a distributor.
     CHECKOUT = u"check_out"
```

Comment:
As per Slack conversation suggest you prepend "distributor_" to these to reduce the possibility of confusion between the two different types of events.